tests/model_registry/conftest.py

```diff
@@ -303,3 +301,69 @@ def generated_schema(model_registry_instance_rest_endpoint: str) -> Any:
         uri="https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml",
         base_url=f"https://{model_registry_instance_rest_endpoint}/",
     )
+
+
+@pytest.fixture(scope="class")
+def dsc_resource_scope_class(admin_client: DynamicClient) -> DataScienceCluster:
+    dsc = DataScienceCluster(name=py_config["dsc_name"])
+    if dsc.exists:
+        return dsc
+    raise ResourceNotFoundError(f"DSC resource {dsc.name} not found")
+
+
+@pytest.fixture(scope="class")
+def updated_dsc_component_state_scope_class(
+    request: FixtureRequest,
+    model_registry_namespace: Namespace,
+    dsc_resource_scope_class: DataScienceCluster,
+) -> Generator[DataScienceCluster, Any, Any]:
+    original_components = dsc_resource_scope_class.instance.spec.components
+    with ResourceEditor(patches={dsc_resource_scope_class: {"spec": {"components": request.param["patch"]}}}):
```

Comment:
if you only run test_model_registry_operator_env, will request.param["patch"] work?

Reply:
Yeah. I don't see why not, and also ran only test_model_registry_operator_env to double check - without any issues.